### PR TITLE
docs: Allow Discharge despite Unbilled Services

### PR DESCRIPTION
**Pull request by ruchamahabal**
Docs for PR: https://github.com/frappe/erpnext/pull/24281

docs-allow-discharge

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md [hunk 1]
```diff
@@ -27,13 +27,19 @@ To view and change the settings, go to:
 
 * **Enable Free Follow-ups**: Many healthcare facilities do not charge for follow up consultations within a time period after the first visit (Patient Registration). Check this if you want to enable free follow-ups. After this is checked, configure the number of free follow-ups (_Patient Encounters in Valid Days_) allowed as well as the time period (_Valid number of days_) for free consultations here.
 
-## 2. Default Healthcare Service Items
+## 2. Inpatient Settings
+
+* **Allow Discharge Despite Unbilled Healthcare Services**: If this is checked then while discharging a patient, the system will not validate unbilled invoices against the patient. Some healthcare facilities discharge the patient to make the room vacant for new admissions and then generate the bill later.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md:32]:
```suggestion
* **Allow Discharge Despite Unbilled Healthcare Services**: Some healthcare facilities discharge the patient to make the room vacant for new admissions and then generate the bill later. If this is checked then while discharging a patient, the system will not validate unbilled invoices against the patient. 
```

File: erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md [hunk 2]
```diff
@@ -27,13 +27,19 @@ To view and change the settings, go to:
...
+
+<img class="screenshot" alt="ERPNext Healthcare" src="{{docs_base_url}}/assets/img/healthcare/inpatient-settings.png">
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md:34]: full image with the checkbox highlighted?

File: erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md [hunk 3]
```diff
@@ -71,7 +77,7 @@ You can enable sending SMS alerts on Patient appointment Booking, Patient Regist
 
 <img class="screenshot" alt="ERPNext Healthcare" src="{{docs_base_url}}/assets/img/healthcare/healthcare_settings_5.png">
 
-## 5. Laboratory SMS ALerts
+## 7. Laboratory SMS ALerts
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md:80]:
```suggestion
## 7. Laboratory SMS Alerts
```

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md (hunk 2 above)
Comment (ruchamahabal) [reply erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md:34]: Actually, we are writing the explanations section-wise so I have added section-wise screenshots.

Comment (ruchamahabal) [reply erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md:34]: Will rectify the images in the next PR.

**Pull request merged**